edge-server/anthos/node-setup-common.sh.tmpl

```diff
@@ -225,3 +225,8 @@ setup_nvidia_container_runtime() {
   fi
 
 }
+
+export_kubeconfig_path() {
```

Comment:
Why do you need these variables there? Do all users need these variables defined? What if the user doesn't use bash?

See [this page](https://blog.flowblok.id.au/2013-02/shell-startup-scripts.html) for more info about where to put these variables according to your use case.

Reply:
remove this for now. will invetisgate further and resumit another PR for this.
this is based on discussion with Markku to let users easier to collect images..etc.